Dockerfile.node

```diff
@@ -1,10 +1,10 @@
-FROM debian:9
+FROM debian:10
```

Comment:
@inductor, let's leave debian:9 as of now because in 10 they changed iptables management and it requires additional configuration in order to work with kube-proxy